website/src/pages/Home/workloads.tsx

```diff
@@ -0,0 +1,290 @@
+import React, { useState, useEffect } from "react";
+import { Typography, Link, Box, Button, Paper, withStyles, Tooltip } from '@material-ui/core';
+import { useTranslation } from "react-i18next";
+import useStyles from './styles';
+import { EXTERNAL_LINKS, VIEW_PORT } from "../../constants";
+import Asciinema from '../../components/Asciinema';
+import { useViewport } from "../../hooks/viewportWidth";
+import { useExternalStyles } from "../../hooks/useExternalStyles";
+import { useScript } from '../../hooks/useScript';
```

Comment:
similar to `useExternalStyles`, can we rename this as `useExternalScript` for consistency?